test/mod/test_api.py

```diff
@@ -103,6 +103,7 @@ def exception(path):
             self.assertIn(field, e, f"Exception needs '{field}'")
         self.assertEqual(e["value"], "osbuild test exception")
         self.assertEqual(e["type"], "ValueError")
+        self.assertIn("exception", e["traceback"])
```

Comment:
This assertion is rock solid! And no, I don't have a better idea ;)

Reply:
I know, I feel bad :/